cli/cmd/play.go

```diff
@@ -118,10 +123,17 @@ func internalPlayModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	os.Setenv("TT_CLI_PLAY_FROM", strconv.FormatUint(playFlags.From, 10))
 	os.Setenv("TT_CLI_PLAY_TO", strconv.FormatUint(playFlags.To, 10))
 
+	iProtoTimestamp, err := util.TsStrToIprotoTs(playFlags.Timestamp)
+	if err != nil {
+		return fmt.Errorf("failed to parse a timestamp: %s", err)
+	} else {
+		os.Setenv("TT_CLI_PLAY_TIMESTAMP", iProtoTimestamp)
+	}
```

Comment:
The same, but more clear.
```suggestion
	if err != nil {
		return fmt.Errorf("failed to parse a timestamp: %s", err)
	}
	os.Setenv("TT_CLI_PLAY_TIMESTAMP", iProtoTimestamp)
```

Reply:
Agree.